doc/deployment/deploy-to-azure-web-app-javase-using-jenkins.md

```diff
@@ -0,0 +1,92 @@
+# Deploy to Azure Web App for Java SE applictions using Jenkins
+
+This tutorial shows you how to deploy a Java SE app to Azure Web App using [Azure App Service Plugin](https://wiki.jenkins.io/display/JENKINS/Azure+App+Service+Plugin).
+Below are the major steps in this tutorial.
+- [Create Azure Container Registry](#create-acr)
```

Comment:
Why do you need the #create-acr section? Content not applicable to deploy to web app on Linux Java SE should be removed.

Reply:
done